spec/intl.html

```diff
@@ -1258,6 +1266,8 @@ <h1>Temporal.Calendar.prototype.eraYear ( _dateOrDateTime_ )</h1>
           <emu-alg>
             1. Let _calendar_ be the *this* value.
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
+            1. If _dateOrDateTime_ does not have an [[ISOYear]] internal slot, then
```

Comment:
I’m not sure the spec asks this about something it doesn’t already know is an Object. It seems intuitive that a primitive would neatly fail this test, and i don’t have a specific change in mind, but i wanted to call it out.

Reply:
That's a good point. Do you think it'd be better as "`If _dateOrDateTime_ is not an Object or does not have an [[ISOYear]] internal slot, then`"? Or would you prefer the more concise version here?

Reply:
I think I'll defer to @bakkot, @syg, and @michaelficarra on this one; there's a few other cases in this PR (including "if it has the slot") where it'd apply as well.

Reply:
I'd probably say `If Type(_dateOrDateTime_) is not Object or _dateOrDateTime_ does not have an [[ISOYear]] internal slot, then`, which is close to the phrasing in [`RequireInternalSlot`](https://tc39.es/ecma262/#sec-requireinternalslot).

Reply:
Wilco, thanks